content/en/docs/tasks/observability/distributed-tracing/configurability/index.md

```diff
@@ -0,0 +1,112 @@
+---
+title: Configurability (Experimental)
+description: How to configure tracing options (experimental).
+weight: 60
+keywords: [telemetry,tracing]
+---
+
+Istio provides advanced capability to configure tracing options as of Istio 1.6.
```

Comment:
nit: i thought i recalled guidance about not putting version strings into docs. Should we just drop "as of Istio 1.6" and expand the next part?  maybe:

> Istio provides the ability to configure advanced tracing options, including sampling rates and span tags. All of these features are considered experimental for this release.